src/backend/distributed/citus--5.2-1--5.2-2.sql

```diff
@@ -0,0 +1,33 @@
+/* citus--5.2-1--5.2-2.sql */
+
+ALTER TABLE pg_dist_partition ADD COLUMN colocationId BIGINT DEFAULT 0;
+
+CREATE SEQUENCE citus.pg_dist_collocation_id
+    MINVALUE 1
+    NO CYCLE;
+
+ALTER SEQUENCE  citus.pg_dist_collocation_id SET SCHEMA pg_catalog;
+
+/* a very simple UDF that only sets the colocation ids the same */
+CREATE OR REPLACE FUNCTION colocate_tables(source_table regclass, target_table regclass)
```

Comment:
I think checking-in such a function could be misleading. Let's remove this from the PR and let users manually update the colocationId for now.
Is that OK for you as well?